issuetracker-api/src/main/java/com/comento/issuetracker/constant/HttpStatusEnum.java

```diff
@@ -0,0 +1,20 @@
+package com.comento.issuetracker.constant;
+
+import lombok.Getter;
+
+public enum HttpStatusEnum {
+    OK(200, "OK"),
+    BAD_REQUEST(400, "BAD_REQUEST"),
+    NOT_FOUND(404, "NOT_FOUND"),
+    INTERNAL_SERER_ERROR(500, "INTERNAL_SERVER_ERROR");
+
+    @Getter
+    int statusCode;
+    @Getter
+    String statusMessage;
```

Comment:
```suggestion
    final int statusCode;
    final String statusMessage;
```

`nit ` final 사용을 권장합니다.
`nit` Getter 는 클래스 레벨에 붙이는 것이 더 좋다고 생각합니다.
`nit` 생성wk eotls `@AllArgsConstructor` 을 활용할 수 있을 것 같아요.

Reply:
@Getter 를 클래스 레벨에 붙일때 얻을 수 있는 이점이 있는지 궁금합니다!

Reply:
- LOC (line of code) 가 줄어듭니다.
- lombok 을 보수적으로 쓰라는 얘기가 많긴 하지만 개인적으로 getter 는 클래스 레벨에서 열어줘도 좋다고 생각 합니다.

---

![image](https://user-images.githubusercontent.com/55722186/199824214-0dd90acc-0634-4add-8fad-0bbcb093ac7a.png)
- `@` 를 사용하는 경우는 꼭 별도 처리를 해주세요. 엄한 계정이 태그 됩니다.